packages/slate/src/interfaces/editor/deleteMerge.ts

```diff
@@ -176,11 +175,5 @@ export const deleteMerge = <V extends Value>(
         voids,
       });
     }
-
-    const point = endRef.unref() || startRef.unref();
-
-    if (options.at == null && point) {
-      select(editor as any, point);
-    }
```

Comment:
@zbeyens Do you remember what the intent of this stanza was?

Reply:
This is a fork from slate. We should sync the diffs.

Reply:
Could we make a check `if only list items selected`, execute `deleteFragmentList` otherwise just `deleteFragment`?

Reply:
That would be better indeed

Reply:
Do you have in mind ways how to check it properly?

Reply:
I've suggested solution in the latest commit